.github/workflows/santilizer.yml

```diff
@@ -0,0 +1,18 @@
+name: Santilizer checks
+
+on: [push, pull_request]
+
+jobs:
+  asan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Envinfo
+        run: npx envinfo
+      - name: ASAN
+        run: |
+          mkdir build
+          cd build && cmake .. -DBUILD_TESTING=ON -DASAN=ON -DCMAKE_BUILD_TYPE=Debug
+          cmake --build .
+          ./uv_run_tests_a
+
```

Comment:
Typo in file name: `s/santilizer.yml/sanitizer.yml` (but I'd probably call it `asan.yml`.)